### DISCUSSION TITLE
수영모드 전환 방법 문의 드립니다.

### DISCUSSION THREAD

**Discussion by limitless** [naverz/zepeto-studio-kor#621] [5. Q&A]
빌드잇에서 제작한 맵을 유니티로 내려받아 작업중인데 물에 들어가는 경우 수영모드로 전환이 안됩니다.
빌드잇에서는 수영모드(애니메이션)이 됩니다.
어떻게 해야 될까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 수영 모드는 아직 지원 전입니다. 
모션V2에서 더블점프가 추가된 것처럼 앞으로도 지속적으로 업데이트 해 나갈 예정입니다. 
감사합니다.
